core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -478,6 +482,8 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
       s"""
         |Options:
         |  --master MASTER_URL         spark://host:port, mesos://host:port, yarn, or local.
+        |  --insecure-skip-tls-verify  Ignore insecure certificates when connection with TLS to the
```

Comment:
I agree with @tnachen.  I don't want to add any more CLI params, especially not niche params such as this one.  Also, there are lots of properties that relate to CLI params, but aren't themselves CLI params, so I don't think that's a reason.